### PR TITLE
Product

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Product/

### CODE REVIEW

File: html/Product/index.html [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+          <label for="amount">QTY</label>
+          <div class="form__container form__amount">
+              <input type="number" min="1" max="10" value="1"> 
+          </div>
+        </div>
+      </form>
+
+      <!-- Reset link -->
+      <a href="#" class="details__reset">
+        clear selection
```
Comment (tsvetkovv) [inline html/Product/index.html]: https://www.w3schools.com/tags/att_button_type.asp
https://www.w3schools.com/tags/att_input_type_reset.asp

File: html/Product/index.html [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+      <!-- Carts -->
+      <button class="logo-section__cart">cart(3)</button>
+    </div>
+
+    <!-- Navbar -->
+    <nav class="navbar">
+      <div class="navbar__actions">
+        <button class="navbar__like">&#9825;</button>
+        <span class="navbar__separator"></span>
```
Comment (tsvetkovv) [inline html/Product/index.html]: please try to implement separators as borders

File: html/Product/index.html [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    </nav>
+  </header>
+
+  <main class="product">
+
+    <!-- Images -->
+    <section class="product__images">
+      
+      <!-- Big image -->
+      <div class="images__big-img"></div>
```
Comment (tsvetkovv) [inline html/Product/index.html]: You can google some [image](https://placeimg.com) [placeholders](https://placeholder.com) and put it there. Just for funny development 😄

Comment (tsvetkovv) [reply html/Product/index.html]: and not just for fun. If you open the site on wide-screen with real images, you see that they are stretched.
image
p.s. is there no limitation for stretching on wide-screens?

File: html/Product/index.html [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+      <div class="details__rate">
+        <div class="rate__stars">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+        </div>
+        <p>3 Review(s)</p>
+        <p>|</p>
```
Comment (tsvetkovv) [inline html/Product/index.html]: border?

File: html/Product/index.html [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+              Free shipping with 4 days delivery
+            </li>
+          </ul>
+      </div>
+
+      <!-- Order form -->
+      <form class="details__form">
+        <!-- Color -->
+        <div>
+          <label for="color">COLOR</label>
```
Comment (tsvetkovv) [inline html/Product/index.html]: The value of the `for` attribute must be the ID of input

File: html/Product/style.css [hunk 6]
```diff
@@ -0,0 +1,544 @@
...
+  margin-top: 10px;
+  opacity: 0.5;
+  border: 1px solid #ccc;
+  overflow: hidden;
+}
+
+.form__color select,
+.form__size select,
+.form__amount input {
+  padding: 10px 0 10px 10px;
```
Comment (tsvetkovv) [inline html/Product/style.css]: why zero? why 130%?

**Review by tsvetkovv — CHANGES_REQUESTED** [review]

File: html/Product/index.html [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+        </div>
+       
+        <!-- Reset -->
+        <button type="reset" class="form__reset">
+          clear selection
+        </button>
+        
+        <div class="form__bookmarks">
+          <!-- Add to cart -->
+          <button class="bookmarks__button-1" type="submit">
```
Comment (tkachukivan) [inline html/Product/index.html]: button-1 and button-2 bad class names, this buttons have a lot in common, different only color/background why not make something as base styles and with modifiers when needed change background and color

**Review by tkachukivan — APPROVED** [review]

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**